kotlinlib/src/mill/kotlinlib/js/KotlinJsModule.scala

```diff
@@ -210,7 +210,7 @@ trait KotlinJsModule extends KotlinModule { outer =>
       outputMode = binaryKindToOutputMode(kotlinJsBinaryKind()),
       irClasspath = Some(compile().classes),
       allKotlinSourceFiles = Seq.empty,
-      librariesClasspath = compileClasspath(),
+      librariesClasspath = upstreamAssemblyClasspath(),
```

Comment:
e.g. what does this have to do with compile time deps?

Reply:
Maybe we could have a new dedicated target, like `runDependencyClasspath` - with all dependency JARs, including JARs made for runtime, but excluding the current module classes. Or maybe we could just rename `upstreamAssemblyClasspath`, given it can have a broader uses.